docs/src/main/asciidoc/_developer-guide/Jakarta_XML_Web_Services/Apache_CXF_integration.adoc

```diff
@@ -670,3 +670,5 @@ disabling CN host name check or not
 
 The vanilla Apache CXF defaults apply when the system properties above
 are not set.
+
+NOTE: References in this document to Interceptors refer to the Jakarta Interceptors unless otherwise noted.
```

Comment:
@RanabirChakraborty , @jimma , please double-check this as well, so that CXF interceptors are not confused with JakartaEE interceptors.

Reply:
@RanabirChakraborty  Again, here is cxf interceptor.